Source/WebCore/dom/Element.h

```diff
@@ -620,7 +621,7 @@ class Element : public ContainerNode {
     void clearPopoverData();
     bool isPopoverShowing() const;
 
-    virtual void handleInvokeInternal(const AtomString&) { }
+    virtual bool handleInvokeInternal(const HTMLFormControlElement*, const AtomString&) { return false; }
```

Comment:
Can we add an argument name here for the `AtomString`? Not obvious what it was/is for.

Why does this function have a return value? The caller doesn’t look at the return value.

Reply:
The return value will be used by future patches. It's used by overrides to check if they should early return. For example dialog will implement this function and call the parents. If it's true (e.g. a popover action occured) it knows it doesn't have any work to do and can just early return.